README.md

```diff
@@ -22,6 +22,27 @@ _Notes_: The crate will look for headers in the `$BORING_BSSL_INCLUDE_PATH/opens
 
 _Warning_: When providing a different version of BoringSSL make sure to use a compatible one, the crate relies on the presence of certain functions.
 
+## Building with a FIPS-validated module
+
+Only BoringCrypto module version ae223d6138807a13006342edfeef32e813246b39, as
+certified with [certificate
+3678](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/3678)
+is supported by this crate. Support is enabled by this crate's `fips-3678` feature.
+
+You must provide and build your own FIPS-validated module in order for this
+crate to link against it (see the instalation instructions in the
+[BoringCrypto FIPS 140-2 Non-Proprietary Security Policy] (https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3678.pdf)
+).
+
+After building the FIPS-validated module, you must build this crate with the
+`BORING_BSSL_PATH`, `BORING_BSSL_INCLUDE_PATH`, and maybe the
```

Comment:
Why is this true? What happens if you don't set these variables?

Reply:
Hmm, ideally we could somehow give a compile error if you try and use a non-FIPS library when the FIPS feature is enabled. Maybe we can have it link to a symbol that only exists in the FIPS version? Or add an assert somewhere?